src/io/utilities/functionSignatures.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Yes, they are created manually.
True, maintaining them is a challenge, if the function signatures change (which should only happen in major revisions, where I would claim, that this effort is worthwile.